README.md

```diff
@@ -38,9 +38,9 @@ The original authors of this code are
 Jasleen Malvai ([@jasleen1](https://github.com/jasleen1)),
 Kevin Lewi ([@kevinlewi](https://github.com/kevinlewi)), and
 Sean Lawlor ([@slawlor](https://github.com/slawlor)).
-To learn more about contributing to this project, [see this document](https://github.com/novifinancial/akd/blob/main/CONTRIBUTING.md).
+To learn more about contributing to this project, [see this document](https://github.com/facebook/akd/blob/main/CONTRIBUTING.md).
```

Comment:
Should we have a complete list of authors here?

Reply:
Yes, good idea!